src/main/webapp/app/exercises/shared/participation/participation.service.ts

```diff
@@ -87,9 +87,9 @@ export class ParticipationService {
     }
 
     cleanupBuildPlan(participation: StudentParticipation): Observable<EntityResponseType> {
-        const copy = this.convertDateFromClient(participation);
+        this.convertDateFromClient(participation);
```

Comment:
If we don't send this via the network, we also don't need to convert anything, right?